spec/features/user_edits_games_spec.rb

```diff
@@ -0,0 +1,49 @@
+require 'rails_helper'
+
+
+feature 'user can edit games' do
+  let(:game) { FactoryGirl.create(:game) }
+
+  scenario 'users can edit any game' do
+    sign_up
```

Comment:
We were thinking that updating the game would be more open to anyone as long as they were logged in. Only deleting would be specific to game.user or the admins.